api/src/main/java/javax/enterprise/inject/spi/builder/InjectionPointConfigurator.java

```diff
@@ -41,51 +37,6 @@
 public interface InjectionPointConfigurator {
 
     /**
-     * Read the InjectionPoint information from the given {@link Field}.
-     * All relevant information is overwritten.
-     *
-     * @param field defining the InjectionPoint.
-     * @return self
-     */
-    InjectionPointConfigurator read(Field field);
```

Comment:
Ok but how would you add it as injectable field then? Note that `InjectionPointConfigurator` has only `addQualifier` methods.